examples/daal/cpp_sycl/makefile_win

```diff
@@ -102,7 +100,7 @@ COPTS=-m64 $(COPTS)
 
 !IF ("$(RES_EXT)"=="lib")
 !IF ("$(threading)"=="sequential")
-DAAL_LIB_T = "$(DAAL_PATH)\onedal_sequential.lib"
+$(error Sequential version is not supported)
```

Comment:
Does this work with Windows nmake?

Reply:
I have concern about message. It sounds like it's not supported by DPC++ interfaces at all, it's not truth. 

"Sequential version is not supported. To run sequential version please refer to 'examples/daal/cpp'"

Reply:
fixed